cli/graph_util.rs

```diff
@@ -276,7 +280,9 @@ impl ModuleGraphCreator {
         loader: None,
       })
       .await?;
-    self.graph_valid(&graph)?;
+    let r = self.graph_valid(&graph);
```

Comment:
This call should filter out "unsupported media type" errors when graph is used for linting